sql_generators/glean_usage/common.py

```diff
@@ -109,6 +110,26 @@ def list_tables(project_id, only_tables, table_filter, table_name="baseline_v1")
     ]
 
 
+@cache
+def _get_pings_without_metrics():
```

Comment:
IMO it'd make a bit more sense to flip the logic and have this be `_get_pings_with_metrics`, to align with the intent of `ping_has_metrics` where it's used.

Reply:
I agree but I went with this because it gets cached so it only caches one tuple instead of 600.  It's a minor optimization though and it can be changed